projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func Execute() {
+	// define a flag of type boolean with the specific properties (name, value, usage)
+	helpFlag := flag.Bool("h", false, "show help")
+
+	// parse the command line flags
+	flag.Parse()
+
+	// print out the value of helpFlag to check
+	// (!) deference the pointer with *
+	// fmt.Println("--- DEBUG helpFlag", *helpFlag)
+
+	// if the "-h" flag is provided
+	if *helpFlag {
+		fmt.Fprintf(os.Stdout, "go-ls help message\n")
+		return
+	}
+
+	// initialise/default the directory path to where go-ls was called from
+	directoryPath := "."
```

Comment:
different os systems might interpret this their own way. Can you look into a way to make it platform agnostic?

hint: it's in the `os` package

Reply:
I think this one `os.Getwd()`

Getwd returns a rooted path name corresponding to the current directory. If the current directory can be reached via multiple paths (due to symbolic links), Getwd may return any one of them.